user-form.html

```diff
@@ -170,7 +170,8 @@ <h2>Account Details</h2>
               type="text"
               id="profile_pic"
               name="profile_pic"
-              placeholder="Link to your profile picture"
+              placeholder="Your profile picture comes from website-static"
```

Comment:
What if we could link the `website-static` repo? (For someone who doesn't know what is `website-static`)

Reply:
Good idea, let's do that as a follow-up feature

This entire www page is quite temporary. We will move it to an Ember app in future

Reply:
I'm sure if its possible or no since it's placeholder (plain text I suppose)

Reply:
Placeholder is just dead text, but we can always provide a link under the input box. I like your idea.

Reply:
Cool ⚡

Reply:
@swarajpure https://github.com/Real-Dev-Squad/website-www/issues/76